src/main/resources/jnt_serverSettingsManageWebProjects/html/serverSettingsManageWebProjects.flow/view.jsp

```diff
@@ -27,6 +27,13 @@
 <fmt:message key="serverSettings.manageWebProjects.noWebProjectSelected" var="i18nNoSiteSelected"/>
 <c:set var="i18nNoSiteSelected" value="${functions:escapeJavaScript(i18nNoSiteSelected)}"/>
 <script type="text/javascript">
+    function stringShortener(input) {
```

Comment:
I see this repeated elsewhere. Might be worth it to refactor to something like stringutil.js and use `template:addResources` taglib. See usage of regexValidation.js pattern here https://github.com/Jahia/serverSettings/blob/90c60ea131b8411ab18802abc61204e0feee55f0/src/main/resources/jnt_serverSettingsMailServerSettings/html/serverSettingsMailServerSettings.flow/showMailSettings.jsp#L20